stdlib/public/Darwin/Compression/Compression.swift

```diff
@@ -338,11 +338,11 @@ public class InputFilter<D: DataProtocol> {
     // Allocate result
     var result = Data(count: count)
 
-    try result.withUnsafeMutableBytes { (dst_ptr: UnsafeMutablePointer<UInt8>) in
```

Comment:
The type annotation was only needed to force the type checker to choose the deprecated method here; please replace with a variable called `buffer`.

Reply:
Fixed